azure/functions/_abc.py

```diff
@@ -88,3 +89,14 @@ class TimerRequest(abc.ABC):
     @abc.abstractmethod
     def past_due(self) -> bool:
         pass
+
+
+class InputStream(io.BufferedIOBase, abc.ABC):
+
+    @abc.abstractmethod
+    def read(self, size=-1) -> bytes:
+        pass
+
+    @abc.abstractmethod
+    def read1(self, size=-1) -> bytes:
```

Comment:
Why do we need `read1`?

Reply:
I was under the false impression that `BufferedIOBase` required `read1` to be implemented.  Removed.